owncloudApp/src/main/res/layout/list_fragment.xml

```diff
@@ -112,16 +112,6 @@
             fab:fab_size="mini"
             fab:fab_title="" />
 
-        <com.getbase.floatingactionbutton.FloatingActionButton
-            android:id="@+id/fab_upload_from_app"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            fab:fab_colorNormal="@color/primary_button_background_color"
-            fab:fab_colorPressed="@color/owncloud_blue"
-            fab:fab_icon="@drawable/ic_import"
```

Comment:
If we delete this FloatingActionButton, maybe we could delete that drawable too, since we won't use it anymore